src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2120,7 +2120,7 @@ From there, simply click on one of the supported files in the File Explorer and
     <value>How to enable</value>
   </data>
   <data name="Oobe_HowToLaunch.Text" xml:space="preserve">
-    <value>How to launch</value>
+    <value>How to open</value>
```

Comment:
where is this used

Reply:
Used in these instances, i'm ok with this
![image](https://github.com/microsoft/PowerToys/assets/1462282/0421fd4e-b3d1-4021-9dab-6ad40f2bb95c)

Reply:
> i'm ok with this

@crutkas Do you mean "with the original" or "with the change"?